### ISSUE TITLE
MozSciLab countdown

### ISSUE THREAD

**Issue by betatim** [everware/everware#5]
Things we need to do before the mozilla science lab blog post. :red_circle: = critical, needs to be done before, :green_heart: = done, :large_orange_diamond: = nice to have
- [x] :green_heart: validate user input so that both https://github.com/... and git@github.com/... work as repo URLs
- [ ] :red_circle: make sure logout button is always visible to escape error 500
- [x] :green_heart: use github oauth so we have names of those abusing the service
- [x] :red_circle: add "beta" to the "everware" name on the landing page
- [ ] :red_circle: make sure all examples run
  - [x] fix dimuon example
  - [ ] fix roofit-tutorial example
- [x] :large_orange_diamond: link back to github organisation from everware pages
- [x] :large_orange_diamond: everware needs a logo
- [ ] :large_orange_diamond: kill idle containers

**Comment by betatim** [comment]
At some point we remove the header which contains a nice logout button from the basic template `page.html`. Should we just re-enable that? This would help with having a logout button "everywhere"

Would mean modding `login.html` so we don't show it there (jinja2 magic). Other reasons against it?

**Comment by betatim** [comment]
@anaderi can you try and figure out how to modify the dimuon-example's container so the `RooFit` stuff works, maybe @ndawe knows?

**Comment by betatim** [comment]
I found [`cull_idle_servers.py`](https://github.com/jupyter/jupyterhub/blob/master/examples/cull-idle/cull_idle_servers.py) which we could use for the idle containers point.

**Comment by ibab** [comment]
We should add a properly styled logout button :)

**Comment by betatim** [comment]
![everware](https://cloud.githubusercontent.com/assets/1448859/9407660/ae3aab90-480b-11e5-96b4-c8e7ea2e2b5a.png)

Yes, I am also an artist :art:

**Comment by ibab** [comment]
:tada: :+1:

**Comment by anaderi** [comment]
I vote for 3rd one

**Comment by ibab** [comment]
I like the ones with an outline, so 2 or 4

**Comment by betatim** [comment]
I vote 3. Decision made :)

**Comment by betatim** [comment]
Mozilla would like to xpost the blog towards the end of the week. While we don't need to have them hassle us I think it is a good "trick" to have a deadline to get things done. Or we decide to give up on the xpost for now, that is also an option.

I have a first version of some JS for input validation in #16

So I think we are pretty close.
